config.go

```diff
@@ -37,7 +37,7 @@ func ParseSSHConfigFile(path string) (map[string]SSHClientOptions, error) {
 
 	// Read lines in reverse order and parse option for each Host section:
 	lines := strings.Split(string(content), "\n")
-	hostExpr := regexp.MustCompile("\\s*Host\\s*=?\\s*(.+)")
+	hostExpr := regexp.MustCompile("\\s*Host\\b\\s*=?\\s*(.+)")
```

Comment:
It think it's better to add the leading `^` to the regexp and so it's exact match of `Host`.

Reply:
something like this?
```
^Host\\b\\s*=?\\s*(.+)
```

Reply:
The one mentioned on https://github.com/crosbymichael/slex/pull/28#discussion_r155523999

Reply:
done, thanks!